include/crm/pengine/internal.h

```diff
@@ -365,11 +365,12 @@ extern void pe__output_node(pe_node_t * node, gboolean details, pcmk__output_t *
 
 void pe__show_node_weights_as(const char *file, const char *function,
                               int line, bool to_log, pe_resource_t *rsc,
-                              const char *comment, GHashTable *nodes);
+                              const char *comment, GHashTable *nodes,
+                              pcmk__output_t *out);
```

Comment:
I'm thinking this should take the data set instead of out to simplify the callers

Reply:
It doesn't appear to do much to simplify the callers - I think it just replaced the pcmk__output_t parameter with a pe_working_set_t parameter.  But sure, easy enough to do.

Reply:
Yep that's what I meant, data_set instead of data_set->priv, to make it more object-oriented from the caller's point of view.